### PR TITLE
UPDATE: first pass of multiple file styling

### PR DESCRIPTION
**Pull request by ianjon3s**
A first pass of styling for multiple file upload.

Example screenshots to follow (left: current, right: updated)

image

image

### CODE REVIEW

File: editor.planx.uk/src/@planx/components/MultipleFileUpload/Public.tsx [hunk 1]
```diff
@@ -140,15 +140,22 @@ function Component(props: Props) {
...
                 key={`subheader-${fileListCategory}-files`}
                 disableGutters
+                sx={{
+                  background: "transparent",
+                  color: "unset",
+                  padding: "1.5em 0 1em",
+                }}
               >
-                {`${capitalize(fileListCategory)} files`}
+                <Typography variant="h4">
```
[the remark this answers is not in the document]
Comment (ianjon3s) [reply editor.planx.uk/src/@planx/components/MultipleFileUpload/Public.tsx:156]: Thanks @DafyddLlyr , that will be very useful.

[the remark this answers is not in the document]
Comment (ianjon3s) [reply editor.planx.uk/src/@planx/components/MultipleFileUpload/Public.tsx]: This is now updated and is passing @DafyddLlyr

**Review by jessicamcinchak — APPROVED** [review]
This looks great - only noticed one minor spacing thing with the question header help & per-file help icons not being evenly right aligned - happy for that to be tweaked here or in a separate second round of updates!
Screenshot from 2023-06-14 13-53-43

**Comment by ianjon3s** [comment]
Thanks @jessicamcinchak , I'll get this merged and pick up in the next styling round, to keep `main` as fresh as possible.

**Pull request merged**